pkg/userdata/generate/generate.go

```diff
@@ -99,6 +101,12 @@ func NewInput(clustername string, masterIPs []string) (input *Input, err error)
 		return nil, err
 	}
 
+	// Create the init token
+	btoken, err := token.NewToken()
```

Comment:
nit: `tok, err := ...` (that is what was used somewhere else, this is just for consistency I can live with this)